picard/tagger.py

```diff
@@ -824,9 +827,8 @@ def cluster(self, objs):
         for name, artist, files in Cluster.cluster(files, 1.0, self):
             cluster = self.load_cluster(name, artist)
             cluster_files[cluster].extend(files)
-        for cluster, files in cluster_files.items():
+        for cluster, files in process_events_iter(cluster_files.items(), steps=2):
```

Comment:
`steps` parameter should be removed with new code

Reply:
Fixed that